onnx_tf/handlers/backend/dynamic_slice.py

```diff
@@ -0,0 +1,71 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("DynamicSlice")
+@tf_func(tf.slice)
+class DynamicSlice(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    input_tensor = tensor_dict[node.inputs[0]]
+    starts = tf.cast(tensor_dict[node.inputs[1]], tf.int64)
+    ends = tf.cast(tensor_dict[node.inputs[2]], tf.int64)
+    l = list(range(starts.shape[0]))
+    axes = tf.cast(tensor_dict[node.inputs[3]], tf.int64) if len(
+        node.inputs) == 4 else tf.constant(l, tf.int64)
```

Comment:
Also `axes`. `starts`, `ends` and `axes` is ensured the same dtype.

Reply:
Thanks for the comments. I wasn't sure if axes, starts, and ends are ensure the same type. Now I know, so I just removed some casts.